Cardgame/admin.py

```diff
@@ -14,4 +14,5 @@ class DeckAdmin(admin.ModelAdmin):
 
     list_display = ('deck_name','number_of_card')
 
-admin.site.register(Deck, DeckAdmin)
\ No newline at end of file
+admin.site.register(Deck, DeckAdmin)
+admin.site.register(Playerscore)
```

Comment:
delete this code because playerscore cannot add or create

Reply:
กรอันนี้กู merge เอง

Reply:
ok krub